torch/utils/tensorboard/summary.py

```diff
@@ -493,8 +493,8 @@ def audio(tag, tensor, sample_rate=44100):
         print('warning: audio amplitude out of range, auto clipped.')
         tensor = tensor.clip(-1, 1)
     assert(tensor.ndim == 1), 'input tensor should be 1 dimensional.'
+    tensor = (tensor * np.iinfo(np.int16).max).astype('<i2')
 
-    tensor_list = [int(32767.0 * x) for x in tensor]
     import io
     import wave
     import struct
```

Comment:
Let's remove these imports since they're not being used anymore.

Reply:
Oh, sorry, well-spotted! I've amended the commit.